shinydashboard/lantern/functions/db_connection.R

```diff
@@ -0,0 +1,30 @@
+# Database connection functions
+
+# Read database connection information from .Renviron file
+# local development: readRenviron("../.env"); db_config$host = "localhost"
+# db_config <- config::get("lantern") 
```

Comment:
is this commented code or just part of the comment?

Reply:
removed the config::get() - no longer used